typhos/utils.py

```diff
@@ -41,6 +42,16 @@
                                       ('attr', 'component', 'signal'))
 DEBUG_MODE = bool(os.environ.get('TYPHOS_DEBUG', False))
 
+# Help settings:
+# TYPHOS_HELP_URL (str): The help URL format string
+HELP_URL = os.environ.get('TYPHOS_HELP_URL', "").strip()
```

Comment:
Should documentation be added here or elsewhere about what this string looks like and options for it? E.g. like your example it can contain fields like `{device.name}`

Reply:
Yes, you're right - let me add some documentation about the documentation (😉) before we move forward with this one.